### ISSUE TITLE
Raspberry Pi 3 B+

### ISSUE THREAD

**Issue by androjet** [lixmk/Wiegotcha#13]
I'm having a plethora of issues both with the auto and manual installs. Is this program incompatible with the 3 B+ or new software?

**Comment by androjet** [comment]
Somewhat fixed, the installer tries to pull wiringPi from drogon, which is no longer active- causing fatal errors. Fixed that and got it to work with exception of the access point.

**Comment by adventuredan** [comment]
were you able to get the AP to work? having this problem myself.

**Comment by androjet** [comment]
Yes! But it's been a while so I can't remeber super clearly what I did. Ended up being some basic AP or SSID setting that I needed to turn off then on manually. I think I got it from some general wireless ap troubleshooting walk-throughs. I'll keep trying to remember

**Comment by adventuredan** [comment]
thanks so much for the reply! i'll see if i can figure it out. also im having the same problem with the wiring pi issue. tried replacing the line to pull from drogon with the wiringpi repo on github and just tried reinstalling and im still getting the wiringpi directory not found error. i forked this repo to make the change but im still a noob with this stuff. any suggestions on what i need to change would be much appreciated! thanks!

**Comment by androjet** [comment]
It was my first project too, big one to start with lol. 

I think when I changed that line of code in the installer I had to make sure it was done on the root user account file of the installer- the code changes I initially made were only affecting the local side copy of the installer, but when I tried to run it again it was pulling the old/flawed version from the root files. 

I think I fully logged into the root user account, found the exact location of the files I wanted, then went back to my normal account and sudo su-ed the changes. Actually editing stuff from the root side is quirky...

My memory is foggy but hopefully that's somewhat intelligible lol.

It was a really fun project, and useful for capturing my own RFID tags. I think pen-testing with it is possible but difficult. I used 2 power sources and my read distance was limited to about 8 inches. The prevalence of NFC also doesn't help. Still, total blast of a project to figure out, and it does actually work!

**Comment by androjet** [comment]
It was a really fun project, and useful for capturing my own RFID tags. 

I think pen-testing with it is possible but difficult. I used 2 power sources and my read distance was limited to about 8 inches. The prevalence of NFC also doesn't help. Still, total blast of a project to figure out, and it does actually work!

**Comment by adventuredan** [comment]
thanks for the tips! i'll see what i can do when i get home. I actually already have a working setup using an RFIDTool from hacker warehouse, and i've been using it successfully for a few years. i get about 2 feet off mine due to using a 24 volt battery pack. 

however the downside of the RFIDTool set up is that the credentials that come in are only in raw binary and hex format, which is fine for use with a proxmark, but it would be nice to identify the protocol for manually writing to a flipper zero. additionally there's no timestamps in the RFIDTool UI so identifying when and where i grabbed a badge from is really difficult.
